serverless.yml

```diff
@@ -7,22 +7,95 @@ plugins:
   - serverless-step-functions
   - serverless-pseudo-parameters
 
+custom:
+  DYNAMO_DB_TABLE: stackJanitorTable
+  DEFAULT_EXPIRATION_PERIOD: 604800 #seconds
+
 provider:
   name: aws
   runtime: nodejs10.x
+  memorySize: 128
+  stage: ${opt:stage, 'dev'}
```

Comment:
could we change default stage to 'development' to match our naming convention for envs?

Reply:
done.